hearpreprocess/nsynth_pitch_kfold.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+"""
+Pre-processing pipeline for NSynth pitch detection, but with 10-fold
+as a way of checking kfold versus 80/10/10 train/val/test split.
+"""
+
+from hearpreprocess.nsynth_pitch import *
+
+generic_task_config["task_name"] = "nsynth_pitch_kfold"
+generic_task_config["split_mode"] = "new_split_kfold"
+# train/val/test is 80/10/10, so 10-fold is identical in split proportions
+generic_task_config["nfolds"] = 10
+
+# generic_task_config["modes"]["5h"] = {"max_task_duration_by_fold": 3600 * 5 * TEST_PERCENTAGE / TRAINVAL_PERCENTAGE}
+# generic_task_config["modes"]["50h"] = {"max_task_duration_by_fold": 36000 * 5 * TEST_PERCENTAGE / TRAINVAL_PERCENTAGE}
+# generic_task_config["modes"]["small"]["max_task_duration_by_fold"] = None
+# del generic_task_config["modes"]["small"]["max_task_duration_by_split"]
+
+# Keep the test length (same as validation) for every fold
```

Comment:
Keeps the test length the same as the non-kfold dataset?

Reply:
Maybe just make it clear that is what is being kept the same. We want to preserve the size of the test split in this kfold version.